types/mapbox-gl/mapbox-gl-tests.ts

```diff
@@ -664,6 +665,10 @@ bounds.extend([
 // $ExpectType LngLatBounds
 bounds.extend([45, 30, 60, 60]);
 
+// controls
```

Comment:
Let's use $ExpectType for new tests. expectType<> is a thing of the past :)

Reply:
I tried to change this. When doing this:

```
// $ExpectType IControl
new mapboxgl.Control();
```

The tests fail with:
```
ERROR: 670:1  expect  TypeScript@4.1 expected type to be:
  IControl
got:
  Control
```

And that's weird because I just made Control implement IControl. Any pointers on how to fix this?

Reply:
Ignore previous comment, using this https://github.com/SamVerschueren/tsd#assertions I figured out the proper way to do this is to use `// $ExpectAssignable IControl`

Reply:
@xivk Well, that is a bit unexpected to me, tbh.

What about this:

// $ExpectType IControl
new mapboxgl.Control() as IControl;

To test the test, try removing "implements IControl" from "Control". Then the test expected to fail.